### PR TITLE
Dron dev

### PR DESCRIPTION
**Pull request by Dron-hihi**

### CODE REVIEW

File: Bon_Voyage/ClientApp/src/views/othersViews/RegisterPage/Register.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React, { Component } from 'react';
+
+
+
+
```
Comment (Karonuk) [inline Bon_Voyage/ClientApp/src/views/othersViews/RegisterPage/Register.jsx]: delete this

File: Bon_Voyage/Controllers/AuthControllers/AuthController.cs [hunk 2]
```diff
@@ -63,5 +64,31 @@ public async Task<IActionResult> Login([FromBody]LoginViewModel model)
                      token = _IJwtTokenService.CreateToken(user)
                  });
         }
+
+        [HttpPost]
```
Comment (Karonuk) [inline Bon_Voyage/Controllers/AuthControllers/AuthController.cs:68]: make a route for this method

File: Bon_Voyage/Controllers/RegistrationControllers/RegistrationController.cs [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                if (userReg != null)
+                {
+                    return BadRequest("Bad Model (цей емейл вже зареєстровано)");
+                }
+
+                if (model.Name == null)
+                {
+                    return BadRequest("Bad Model (поле вводу пусте)");
+                }
+                else
```
Comment (Karonuk) [inline Bon_Voyage/Controllers/RegistrationControllers/RegistrationController.cs]: this check should be on a front

File: Bon_Voyage/Controllers/RegistrationControllers/RegistrationController.cs [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+                if (model.Telefon == null)
+                {
+                    //return BadRequest("Bad Model (номер телефону не вказано)");
+                }
+
+                if (model.DateofBirth == null)
+                {
+                    //return BadRequest("Bad Model (номер телефону не вказано)");
+                }
```
Comment (Karonuk) [inline Bon_Voyage/Controllers/RegistrationControllers/RegistrationController.cs]: this validation on the frontend part

File: Bon_Voyage/Controllers/RegistrationControllers/RegistrationController.cs [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                else
+                {
+                    foreach (var error in resault.Errors)
+                    {
+                        ModelState.AddModelError(string.Empty, error.Description);
+                    }
+                }
+            }
+            return null;
```
Comment (Karonuk) [inline Bon_Voyage/Controllers/RegistrationControllers/RegistrationController.cs:118]: should to return badrequest

**Review by Maksym-Rohach — APPROVED** [review]

**Review by Maksym-Rohach — APPROVED** [review]

**Pull request merged**